tests/pip_install/whl_library/generate_build_bazel_tests.bzl

```diff
@@ -219,6 +249,74 @@ py_binary(
 
 _tests.append(_test_with_entry_points)
 
+def _test_group_member(env):
+    want = """\
+load("@rules_python//python:defs.bzl", "py_library", "py_binary")
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "dist_info",
+    srcs = glob(["site-packages/*.dist-info/**"], allow_empty = True),
+)
+
+filegroup(
+    name = "data",
+    srcs = glob(["data/**"], allow_empty = True),
+)
+
+filegroup(
+    name = "_whl",
```

Comment:
Why do we need this label here?

Reply:
In this example, this is the filegroup which `@pypi__groups//:qux_whl` depends on, same as `@pypi__groups//:qux_pkg` depends on `:_pkg`. See https://github.com/bazelbuild/rules_python/pull/1166/files#diff-ecdc677552e2995c1c31461205f3ce1d875bcafc15453b568644f77c3d37d7d8R29 for the tests on that side.